docs/modules/user-guide/partials/proc_devfile-registry.adoc

```diff
@@ -0,0 +1,45 @@
+[id="proc_devfile-registry_{context}"]
+= Devfile registry
+
+[role="_abstract"]
+A devfile registry is a service that stores and provides devfile stacks to Kubernetes developer tools like odo, Eclipse Che and the OpenShift Developer Console. Therefore, you can access devfiles through the devfile registry.  Devfile registries are based on the Oracle Cloud Infrastructure (OCI) Specification, and devfile stacks are stored as OCI-artifacts in the registry.
+
+Each devfile stack corresponds to a specific runtime or framework, such as,  NodeJS, Quarkus, and Wildfly. A devfile stack also contains resources, such as, a devfile.yaml, logo, and outerloop resources. These devfile stacks give developers a template so they can get started developing cloud-native applications.
+
+== Devfile registry components
+
+A devfile registry has two components: a devfile index server and an OCI registry server.
+
+.Devfile index container
+
+The devfile index container image does the following actions:
+
+* Uses the index.json file to obtain metadata about the stacks and samples and hosts this metadata in the registry for tools to consume.
+* Bootstraps the OCI registry with the devfile stacks.
+* Provides an API to interact with the OCI registry and to retrieve devfile stacks.
+
+.OCI registry
+
+The devfile stacks in a devfile registry are stored in an OCI-compatible artifact registry. The artifact registry is based on the reference implementation of an OCI registry.
+
+When a devfile registry is deployed, the stacks belonging to the devfile registry are pushed to the OCI registry as multi-layer artifacts.
+
+== Types of devfile registries
+
+As a developer, you can use multiple types of devfile registries to access devfile stacks and tools.
+
+.Public community registry
+
+The public community registry provides stacks for various community tools to consume. Examples include: NodeJS, Quarkus, and Open Liberty.
```

Comment:
```suggestion
The public community registry provides stacks for various community tools to consume, such as NodeJS, Quarkus, and Open Liberty.
```
IBM SG: "Use a colon after an _independent clause_ to introduce an inline list."